Stripe/STPPaymentCardTextField.m

```diff
@@ -638,7 +638,7 @@ behaves identically to user entering text (and will have the same forwarding
      */
     STPFormTextField *originalSubResponder = self.currentFirstResponderField;
 
-    self.internalCardParams = cardParams;
+    self.internalCardParams = [cardParams copy];
     [self setText:cardParams.number inField:STPCardFieldTypeNumber];
```

Comment:
might be worth adding a comment here highlighting that it's important to use `cardParams` and not `internalCardParams` for the `sets`